### PR TITLE
#640 cleaning Code Style

### PR DESCRIPTION
**Pull request by elcore**
This is a follow up for #641, the fix experienced after #640 

I hope you are satisfied :smile:

### CODE REVIEW

File: caddy/https/https.go [hunk 1]
```diff
@@ -401,21 +401,21 @@ var (
...
+// Possible types for a key.
 const (
-	Ecc224  KeySize = 224
-	Ecc256          = 256
-	Rsa2048         = 2048
-	Rsa4096         = 4096
+	RSA2048		= acme.RSA2048
+	RSA4096		= acme.RSA4096
+	EC256		= acme.EC256
+	EC384		= acme.EC384
```
Comment (mholt) [inline caddy/https/https.go]: We might not need these. Why not just use the values from the `acme` package directly?

File: caddy/https/https.go [hunk 2]
```diff
@@ -401,21 +401,21 @@ var (
...
 
-// rsaKeySizeToUse is the size to use for new RSA keys.
+// KeyTypeToUse is the type to use for new keys.
 // This shouldn't need to change except for in tests;
 // the size can be drastically reduced for speed.
-var rsaKeySizeToUse = Rsa2048
+var KeyTypeToUse = RSA2048
+
+// KeySize should be equal to KeyTypeToUse: KeyTypeToUse = 4096 -> KeySize = 4096
+var KeySize = 2048
```
Comment (mholt) [inline caddy/https/https.go]: This is a little quirky, to maintain both variables to keep them in sync. We should have a key generation function that can switch on the different key types.

File: caddy/https/user.go [hunk 3]
```diff
@@ -104,7 +104,7 @@ func saveUser(user User) error {
 // instead. It does NOT prompt the user.
 func newUser(email string) (User, error) {
 	user := User{Email: email}
-	privateKey, err := rsa.GenerateKey(rand.Reader, rsaKeySizeToUse)
+	privateKey, err := rsa.GenerateKey(rand.Reader, KeySize)
```
Comment (mholt) [inline caddy/https/user.go]: While we're at it, I'd like to default to ECC keys for user accounts.

File: caddy/https/https.go (hunk 1 above)
Comment (elcore) [inline caddy/https/https.go]: Sure, no problem at all

File: caddy/https/user.go (hunk 3 above)
Comment (elcore) [inline caddy/https/user.go]: You are right, but at the moment we only support RSA Private Keys.

I am working on it

File: caddy/https/https.go (hunk 2 above)
Comment (elcore) [inline caddy/https/https.go]: You are right, that is temp.

**Comment by mholt** [comment]
Closing this in favor of #652.

**Pull request closed without merging**